app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/checkemail/CheckEmailScreenTest.kt

```diff
@@ -63,6 +67,12 @@ class CheckEmailScreenTest : BaseComposeTest() {
         }
     }
 
+    @Test
+    fun `login button click should send LoginTap action`() {
+        mutableEventFlow.tryEmit(CheckEmailEvent.NavigateBackToLanding)
+        TestCase.assertTrue(onNavigateToLandingCalled)
```

Comment:
Can we use `Assert` and we also try to import the functions:

```kotlin
import org.junit.Assert.assertTrue
```

Reply:
oop